mods/stairs/init.lua

```diff
@@ -136,7 +136,6 @@ function stairs.register_stair(subname, recipeitem, groups, images, description,
 	end
 end
 
-
```

Comment:
Although there's a mix of 1 and 2 newline gaps we prefer 2 newline for clarity. No need to make them all 2 newline but best not remove these lines 139 and 256.